src/atmos_spectral/driver/solo/mixed_layer.F90

```diff
@@ -757,10 +757,12 @@ subroutine albedo_calc(albedo_inout,Time)
 
 if(update_albedo_from_ice) then
 
-    where(ice_concentration.gt.ice_concentration_threshold) 
-        albedo_inout=ice_albedo_value
+    where(ice_concentration.gt.ice_concentration_threshold)
+      albedo_inout=ice_albedo_value
     end where
 
+    !albedo_inout = albedo_inout*(1.0-ice_concentration) + ice_albedo_value*ice_concentration
+
```

Comment:
Uncommented version as in present master (step-wise function): trip tests pass
Commented version as in original simcloud branch (ramp function): realistic continents trip tests fail